conf/sample.ini

```
@@ -1188,6 +1188,11 @@
 # Enable the news feed section
 ; news_feed_enabled = true
 
+#################################### Query #############################
+[query]
+# Set the number of data source queries that can be executed concurrently in mixed queries. Default is the number of CPUs.
+;concurrent_query_limit = 1
```

nit; I think that this should comply with the value in the defaults.ini and the comments above
```suggestion
;concurrent_query_limit =
```